### PR TITLE
update institution profile for KU_SUND_DANGPU

### PR DESCRIPTION
**Pull request by AdrijaK**
---
name: Updated Config
about: Updated cluster config
---

Please follow these steps before submitting your PR:

- [x] If your PR is a work in progress, include `[WIP]` in its title
- [x] Your PR targets the `master` branch
- [x] You've included links to relevant issues, if any
- [x] Requested review from @nf-core/maintainers and/or #request-review on slack

Steps for adding a new config profile:

- [x] Add your custom config file to the `conf/` directory
- [x] Add your documentation file to the `docs/` directory
- [x] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
- [x] Add your custom profile to the `README.md` file in the top-level directory
- [x] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`

### CODE REVIEW

File: docs/ku_sund_dangpu.md [hunk 1]
```diff
@@ -38,20 +37,41 @@ mkdir $NXF_TEMP
...
+nextflow run nf-core/rnaseq  -r 3.10.1 -profile ku_sund_dangpu --outdir <name-of-output-directory> --input <name-of-input-csv-file>
 ```
 
 ## Notes
 
-Note that normally on dangpu server you are required to run resource-intensive commands with slurm, but at ku_sund_dangpu profile we have pre-configured slurm to be the resource manager within the `ku_sund_dangpu profile`. Just make sure that the pipeline is run within a tmux session or within a screen session.
+Note that normally on dangpu server you are required to run resource-intensive commands with slurm, but at ku_sund_dangpu profile we have pre-configured slurm to be the resource manager within the `ku_sund_dangpu profile`. Just make sure that the pipeline is run within a tmux session or within a screen session. 
+
+Start a new session and enter the vitual screen named `mysessionname`:
+```
```
Comment (mashehu) [inline docs/ku_sund_dangpu.md:59]:
```suggestion
```bash
```
To get a bit nicer syntax highlighting on the [configs page](https://nf-co.re/configs/ku_sund_dangpu)

File: docs/ku_sund_dangpu.md [hunk 2]
```diff
@@ -38,20 +37,41 @@ mkdir $NXF_TEMP
...
+
+# load modules
+# define variables
+# run nfcore pipeline here
+```
+
+Detach a session use control+B and then press d.
+
+To attach a previously detached session again:
+```
```
Comment (mashehu) [inline docs/ku_sund_dangpu.md:70]:
```suggestion
```bash
```

File: docs/ku_sund_dangpu.md [hunk 3]
```diff
@@ -38,20 +37,41 @@ mkdir $NXF_TEMP
...
+tmux atttach-session -t mysessionname
+```
+
+To exit a tmux session when the job is done:
+```
```
Comment (mashehu) [inline docs/ku_sund_dangpu.md:75]:
```suggestion
```bash
```

**Comment by matq007** [comment]
@nf-core-bot fix-linting

**Comment by matq007** [comment]
@nf-core-bot fix linting

**Review by matq007 — APPROVED** [review]
LGTM

**Pull request merged**